hwy/targets.h

```diff
@@ -148,6 +148,13 @@ static inline HWY_MAYBE_UNUSED const char* TargetName(int64_t target) {
       return "RVV";
 #endif
 
+#if 1  // TODO: arch name
```

Comment:
arch name is LOONGARCH. In addition, there seems to be no relevant define in hwy/detect_compiler_arch.h.

Reply:
Right, thanks. Just to confirm, you'd like HWY_ARCH_LOONGARCH?